applications/ssotap/secrets-idfdev.yaml

```diff
@@ -0,0 +1,11 @@
+"google_creds.json":
+  description: >-
+    Google service account credentials used to write async job output to
+    Google Cloud Storage.
+pgpassword:
+  description: >-
+    Password to external PostgreSQL server that contains the Solar System
+    Objects data.
+uwspassword:
+  description: >-
+    Password to external UWS PostgreSQL server
```

Comment:
The existing naming scheme for secrets is all over the map, but I'm trying to standardize on dash-separated names since they're a bit more human-readable. How about `uws-db-password` for this? (Or `uws-database-password` if you want to be more verbose.)

Reply:
Sure that makes sense. I've changed it to uws-db-password. Was wondering if I should change "pgpassword" from the ssotap secrets while I'm here, but maybe I'll leave that for now